node/node.go

```diff
@@ -590,6 +602,10 @@ func (node *Node) Start() error {
 						return false
 					}
 
+					if ignore {
+						return true
```

Comment:
Is it true that ignored message will trigger error `errConvertToValidMessage`?

Reply:
you are right, should ignore it.